src/App.css

```diff
@@ -1,38 +1,38 @@
 .App {
-  text-align: center;
+	text-align: center;
+	/* background-color: #282c34; */
+	color: white;
 }
 
 .App-logo {
-  height: 40vmin;
-  pointer-events: none;
+	height: 40vmin;
+	pointer-events: none;
 }
 
 @media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+	.App-logo {
+		animation: App-logo-spin infinite 20s linear;
+	}
 }
 
 .App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+	/* min-height: 100vh; */
+	display: flex;
```

Comment:
yay flex box! I love flex box :D

Reply:
I will learn to love flex box. I've added formatting as low down on the TODO as I could justify. Perhaps I'll open a branch just for experimenting with CSS as a way to positively procrastinate.